internal/changelog/changelog.go

```diff
@@ -129,8 +157,17 @@ func (c *Changelog) GenerateChangelog(templateConfig shared.ChangelogTemplateCon
 		}
 	}
 
+	authors := map[string]int{}
+
 	for _, commits := range analyzedCommits {
 		for _, commit := range commits {
+
+			_, ok := authors[commit.Commit.Author]
+			if !ok {
+				authors[commit.Commit.Author] = 0
+			}
+			authors[commit.Commit.Author] = authors[commit.Commit.Author] + 1
+
```

Comment:
Why do you count the contributor/commit? If we do not care about it we could use an empty struct here

Reply:
At the beginning i showed the amounts of commits per author. But removed it in the template. I will remove it here too